### DISCUSSION TITLE
Created Roomba (s9) map and card, but 'Bin present' and 'Bin full' are "NaN"

### DISCUSSION THREAD

**Discussion by douginoz** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#620] [Q&A]
I have 2 roombas installed but I have only created one card for one of the roombas.  Everything works fine, but the "Bin Present" and "Bin full" shows "NaN".  If I click on one of them and bring up the vacuum's details, the Attributes drop-down shows
```

Bin present     true
Bin full          true
```

The entities available for this robot are:
```

vacuum.roomba2
sensor.roomba2_battery_level
binary_sensor.roomba2_bin_full
```
These states persist regardless of restarts or using the card / operating the roomba.

**Comment by PiotrMachowski** [comment]
Can you probide vacuum attributes from the "State attributes" textfield in dev tools (it is filled after clicking entity id)?

**Reply by douginoz** [reply]
```
fan_speed_list:
  - Automatic
  - Eco
  - Performance
battery_level: 100
battery_icon: mdi:battery-charging-100
fan_speed: Automatic
software_version: soho+22.29.6+2022-12-01-82f3372a65c+Firmware-Build+2321
status: Charging
bin_present: true
bin_full: true
friendly_name: Roomba2
supported_features: 13308
```

**Reply by avhoboken** [reply]
I have exactly the same issue, did you manage to solve it?
Thanks!

**Reply by douginoz** [reply]
No but I forgot to follow up and do what the author asked. I'm overseas for a month so I can't try it but if you check in this thread you'll see that there's some additional tests you can do that might provide the  data the author needs to identify the problem

**Reply by DataheadMarie** [reply]
Remove the "precision" line. Worked for me!
